libs/core/langchain_core/outputs/llm_result.py

```diff
@@ -16,7 +18,9 @@ class LLMResult(BaseModel):
     wants to return.
     """
 
-    generations: List[List[Generation]]
+    generations: List[
+        List[Union[Generation, ChatGeneration, GenerationChunk, ChatGenerationChunk]]
```

Comment:
ooc why's Generation not good enough anymore, do things get coerced to Generation?

Reply:
pydantic 2 is stricter with respect to union discrimination